.github/workflows/upload-nextjs-integration-test-results.yml

```diff
@@ -41,13 +41,13 @@ jobs:
           echo "SHA_SHORT=$(git rev-parse --short HEAD)" >> $GITHUB_ENV
           echo "RESULT_SUBPATH=$(if ${{ inputs.is_main_branch }}; then echo 'main'; else echo ${{ env.NEXTJS_VERSION }}; fi)" >> $GITHUB_ENV
 
-      # Rename test results to `${date}-${nextjs-version}-${sha-short}.json`.
+      # Copy test results to `${date}-${nextjs-version}-${sha-short}.json`.
       # If workflow is not coming from main branch update, then we need to move test results to subpath `/${nextjs-version}`.
       - name: Congifure subpath
         run: |
           echo "Configured test result subpath for ${{ env.RESULT_SUBPATH }} / ${{ env.NEXTJS_VERSION }} / ${{ env.SHA_SHORT }}"
           mkdir -p test-results/${{ env.RESULT_SUBPATH }}
-          mv -v test-results/main/nextjs-test-results.json test-results/${{ env.RESULT_SUBPATH }}/$(date '+%Y%m%d%H%M')-${{ env.NEXTJS_VERSION }}-${{ env.SHA_SHORT }}.json
+          cp -v test-results/main/nextjs-test-results.json test-results/${{ env.RESULT_SUBPATH }}/$(date '+%Y%m%d%H%M')-${{ env.NEXTJS_VERSION }}-${{ env.SHA_SHORT }}.json
```

Comment:
git pull occurs after this step if I recall, will it gracefully preserve the changes or error out?

Reply:
Oh, I didn't realize. We could just reorder them, I think?

Reply:
Probably yes? can see what happens in a day I guess.